src/sites/twitch-twilight/modules/css_tweaks/styles/player-event-bar.scss

```diff
@@ -0,0 +1,3 @@
+.channel-page .live-event-banner-ui__header {
+	display: none !important;
```

Comment:
The css_tweaks module provides a specific helper for applying `display: none !important` to rules. Please use `.toggleHide(...)` rather than a custom block of CSS.